testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -313,22 +321,106 @@ impl RewardSetEventPayload {
     }
 }
 
+#[cfg(test)]
+static TEST_EVENT_OBSERVER_SKIP_RETRY: std::sync::Mutex<Option<bool>> = std::sync::Mutex::new(None);
+
 impl EventObserver {
-    pub fn send_payload(&self, payload: &serde_json::Value, path: &str) {
+    fn init_db(db_path: &str) -> Result<Connection, db_error> {
+        let conn = Connection::open(db_path)?;
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS pending_payloads (
```

Comment:
I don't think so, but does this database need to be version controlled? I assumed not because it's best effort resending payloads across restarts and if they are lossed due to version incompatibility it doesn't matter right?

Reply:
Good point, but I'd lean towards keeping it simple, treating it as a throw away database. Like if we need to change the schema, then we can just change the name.